### ISSUE TITLE
[Feature]: A simple lock button for X/Y/Z Scripts. 

### ISSUE THREAD

**Issue by 4joeknight4** [vladmandic/sdnext#2231]
### Feature description

I was wondering if it would be possible to be able to make a "lock" checkbox for the scripts section (mostly X/Y/Z Grids). 

I mix and train my own models and reuse old images to check the differences between merges, epochs, etc. When I pull the metadata from the Process Image tab and transfer it to the From Text tab, the scripts section reverts back to null. Re-selecting all the checkpoints I applied in the last image generation is an inconvenience (I know, I'm lazy and I have many models). 

Just an idea I had and figured would be a nice feature. 

Thanks!

### Version Platform Description

win10, firefox, usually the latest commit.

**Comment by vladmandic** [comment]
this is pretty isolated use-case with non-trivial work associated with it. 
contributions would be welcome, but otherwise i don't see this making it to priority list.

**Comment by 4joeknight4** [comment]
Fair enough. I do agree that it is a very isolated ise-case. Thanks though!
